api/subscriptions/views.py

```diff
@@ -277,26 +276,6 @@ def update(self, request, *args, **kwargs):
 
         else:
             return super().update(request, *args, **kwargs)
-=======
-        ret = super().update(request, *args, **kwargs)
-        obj = self.get_object()
-        # Copy global_reviews subscription changes to new_pending_submissions subscriptions [ENG-9666]
-        if obj.event_name == 'global_reviews':
-            user = obj.user
-            qs = NotificationSubscription.objects.filter(
-                event_name='new_pending_submissions',
-            ).filter(
-                Q(none=user) |
-                Q(email_digest=user) |
-                Q(email_transactional=user),
-            ).distinct()
-            for subscription in qs:
-                data = {**request.data, 'id': subscription._id}
-                serializer = self.get_serializer(subscription, data=data)
-                serializer.is_valid(raise_exception=True)
-                self.perform_update(serializer)
-        return ret
->>>>>>> upstream/hotfix/25.18.1
```

Comment:
This part may have come from https://openscience.atlassian.net/browse/ENG-9666. Do we no longer need this fix since it's deprecated in NR? Or do we actually have to reimplement this on top of NR?